types/detect/detect.go

```diff
@@ -49,6 +48,6 @@ type SignalSourceComplete string
 //Finding is the main output of a signature. It represents a match result for the signature business logic
 type Finding struct {
 	Data        map[string]interface{}
-	Context     Event
+	Payload     interface{} //Payload is the event's payload that caused the Finding
```

Comment:
Sorry for catching it late, but I think we may add protocol.Event here. Otherwise someone who's processing findings won't know the ContentType of the Payload.